target/linux/armsr/armv8/config-6.6

```diff
@@ -549,6 +549,11 @@ CONFIG_PHY_NS2_USB_DRD=y
 CONFIG_PHY_RCAR_GEN3_PCIE=y
 CONFIG_PHY_RCAR_GEN3_USB2=y
 CONFIG_PHY_RCAR_GEN3_USB3=y
+CONFIG_SERIAL_SH_SCI=y
+CONFIG_SERIAL_SH_SCI_NR_UARTS=18
+CONFIG_SERIAL_SH_SCI_CONSOLE=y
+CONFIG_SERIAL_SH_SCI_EARLYCON=y
+CONFIG_SERIAL_SH_SCI_DMA=y
```

Comment:
This is already activated in the `target/linux/armsr/armv8/config-6.6` configuration file. no need to add it twice.

Reply:
Hi @hauke 
Thanks. Removed these settings in config-6.6

Reply:
Hi @hauke 
I've rebased this branch now.
The config-6.1 removed and hence the changes not required.
I've kept the Ethernet module updates here
Please merge
Thanks.